### ISSUE TITLE
Head position change - Harden transform was applied for volume rendering but not for scan slices

### ISSUE THREAD

**Issue by KarlaCarpio** [Slicer/Slicer#7800]
## Summary

I altered the head position of a CBCT using the Transforms and Hardened the transform. Resulted in the change of the position of the volume rendering display, but the scan axes are showing the original position (Photo 1)
If I click in reformat in each axe window, it will change the position of the slices to the transformed desired position (same as volume rendering). (Photo 2, 3)
The first step of my analysis involve performing the head position alteration, therefore, I need to correct this step please. 

## Steps to reproduce

1. Loaded an original CBCT scan (Photo 1)
2. Obtained the 3D view with Volume Rendering extension
3. Using Transform extension, created a new linear transform, moved the scan from the transformable to the transformed box, did the position modifications.
6. Moved the scan from the transformed to transformable to the box, which Hardened the Transform
7. Saved the scan file and close the Scene
9. Reopened the scan and obtained a 3D view (Volume Rendering tool): volume rendering shows the changed position, but the scan does not show the same position (Photo 1)
10. When I reformat the Reformat in each window (axial/coronal/sagittal) the orientation improves (Figure 2, 3)

Using the reformat buttons is not an option since it can compromise future measurements that I need to perform. 
Also, in the past I use to solve this problem with the tool "Apply Matrix", but it is not available anymore :(
![1](https://github.com/Slicer/Slicer/assets/138545203/e416c416-83bb-415a-a56e-cc6d71b37992)
![2](https://github.com/Slicer/Slicer/assets/138545203/c046998c-f233-4dc5-b7c7-7710fb8db3c8)
![3](https://github.com/Slicer/Slicer/assets/138545203/f0bc09e5-f43d-47e2-beb4-74955acedab7)

## Environment
- Slicer version: Slicer-5.6.2-2023
- Operating system: Windows 

Please @lassoan Any Suggestion?

**Comment by lassoan** [comment]
It is up to you if you want to snap views to the volume axes or you want to keep the current slice orientations (that is initialized by default to axial, sagittal, coronal): go to `Data` module, right-click on the eye icon of the volume and check or uncheck  `Reset view orientation on show`.

![image](https://github.com/Slicer/Slicer/assets/307929/4471c659-09e2-4c7c-b84f-43c017a61807)

> in the past I use to solve this problem with the tool "Apply Matrix", but it is not available anymore

What is the "Apply Matrix" tool? Where was it available?

The issue tracker is for tracking resolution of confirmed issues, so next time you have questions please post it to the [Slicer forum](https://discourse.slicer.org).

**Comment by KarlaCarpio** [comment]
Hello @lassoan 
Thank you very much for your response! I will follow the procedure at the university. 

"Apply Matrix", was an extension used to apply the matrix created after doing modifications in the Transform Extension:
Steps to apply the Matrix:
1. Loaded an original CBCT scan
2. Obtained the 3D view with Volume Rendering extension
3. Using Transform extension, created a new linear transform, and modified the position. 
4. Save the Matrix in .tmf format (Photo 1)
5. Close Scene
6. Loaded an original CBCT scan + Matrix 
7. Using the "Apply Matrix" extension, applied the matrix to the original scan (Photo 2)
8. Saved new Scan as oriented (it changes the position of the scan using as reference the position of the Matrix) (Photo 3)

Note: I did not harden the Transform when using the "Apply Matrix" extension.

However, I can no longer install "Apply Matrix" in 5.5  and newer. Maybe it was removed since hardening the transform will perform the same function?



![1_Saving Matrix](https://github.com/Slicer/Slicer/assets/138545203/8c2a23f7-8315-4df8-ba85-3901fd02ecd3)
![2_Extension Apply Matrix](https://github.com/Slicer/Slicer/assets/138545203/cd1fed5e-d02e-4601-97b6-c22ee61c7161)
![3_After applying the matrix Saving New Oriented volume](https://github.com/Slicer/Slicer/assets/138545203/154a82ab-e9c2-4057-a3fa-f8fd4bcae87c)

I will post in the Forum next time. Thank you!
